setup.py

```diff
@@ -106,7 +106,7 @@ def build_extension(self, ext: CMakeExtension):
 requirements = [
     "ninja", 
     "numpy",
-    "pennylane>=0.15",
+    "pennylane>=0.19",
 ]
```

Comment:
Hi @mlxd, just a fly-by comment: once Lightning supports Python 3.10, this can also be marked on a project level. To do that, just an extra line is required in this file on line 151, to show the Python 3.10 compatibility.

Reply:
Great, thanks for the tip @antal !